### PR TITLE
Add Orioledb Alpha Blog Post

### PR DESCRIPTION
**Pull request by olirice**
Adds the LW13 OrioleDB Public Alpha blog post

- TODO:
- [x] remove "GA Week / Build Stage" from the OG image
- [x] add a link to the orioledb.com companion blog post
- [x] add an image showing how to access the oriole image from database.new (pending FE work)

### CODE REVIEW

**Review by fsansalvadore — APPROVED** [review]

**Comment by w3b6x9** [comment]
@fsansalvadore can you help to center the logo in image?

Screenshot 2024-11-29 at 17 51 01

File: apps/www/_blog/2024-12-01-orioledb-launch.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: 'OrioleDB Public Alpha'
+description: 'Launching OrioleDB public alpha'
```
Comment (w3b6x9) [inline apps/www/_blog/2024-12-01-orioledb-launch.mdx:3]:
```suggestion
description: 'Launching OrioleDB Public Alpha'
```

File: apps/www/_blog/2024-12-01-orioledb-launch.mdx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+thumb: 2024-12-01-orioledb-release/thumb.png
+categories:
+  - engineering
+tags:
+  - supabase-engineering
+date: '2024-12-01'
+toc_depth: 3
+---
+
+# OrioleDB public alpha
```
Comment (w3b6x9) [inline apps/www/_blog/2024-12-01-orioledb-launch.mdx:15]:
```suggestion
# OrioleDB Public Alpha
```

File: apps/www/public/rss.xml [hunk 3]
```diff
@@ -5,9 +5,16 @@
...
       <description>Latest news from Supabase</description>
       <language>en</language>
-      <lastBuildDate>Wed, 13 Nov 2024 00:00:00 -0700</lastBuildDate>
+      <lastBuildDate>Fri, 29 Nov 2024 00:00:00 -0700</lastBuildDate>
       <atom:link href="https://supabase.com/rss.xml" rel="self" type="application/rss+xml"/>
       <item>
+  <guid>https://supabase.com/blog/orioledb-launch</guid>
+  <title>OrioleDB Public Alpha</title>
+  <link>https://supabase.com/blog/orioledb-launch</link>
+  <description>Launching OrioleDB public alpha</description>
```
Comment (w3b6x9) [inline apps/www/public/rss.xml:14]:
```suggestion
  <description>Launching OrioleDB Public Alpha</description>
```

File: apps/www/_blog/2024-12-01-orioledb-launch.mdx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  - engineering
+tags:
+  - supabase-engineering
+date: '2024-12-01'
+toc_depth: 3
+---
+
+# OrioleDB Public Alpha
+
+Today, we're releasing the public alpha of [OrioleDB](https://www.orioledb.com/) on the Supabase platform.
```
Comment (w3b6x9) [inline apps/www/_blog/2024-12-01-orioledb-launch.mdx:17]:
```suggestion
Today, we're releasing the Public Alpha of [OrioleDB](https://www.orioledb.com/) on the Supabase platform.
```

File: apps/www/_blog/2024-12-01-orioledb-launch.mdx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+This initial release is a Public Alpha and you should _not_ use it for Production workloads. The release comes with several limitations:
+
+- The release is restricted to Free organizations. You will not be able to upgrade OrioleDB projects to larger instance sizes. If you want to run OrioleDB on a larger instance we suggest following the [Getting Started](https://www.orioledb.com/docs/usage/getting-started) guide on OrioleDB’s official website.
+- Index support is restricted to the Postgres default B-Tree index type. Other indexs like GIN/GiST/BRIN/Hash, and pgvector's HNSW/IVFFlat are not supported.
+- No support for Supabase Realtime
+
+### Should you use it today?
+
+At this stage, the goal of adding OrioleDB to the platform is to make it easier for testers to give feedback. If you’re running production workloads, stick to the standard options available.
```
Comment (w3b6x9) [inline apps/www/_blog/2024-12-01-orioledb-launch.mdx:35]:
```suggestion
At this stage, the goal of adding OrioleDB to the platform is to make it easier for testers to give feedback. If you’re running Production workloads, stick to the standard options available.
```

File: apps/www/_blog/2024-12-01-orioledb-launch.mdx [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+You can read more about OrioleDB [here](https://www.orioledb.com/blog/orioledb-beta7-benchmarks) and learn why you might choose it over the default Postgres storage engine.
+
+### Limitations
+
+This initial release is a Public Alpha and you should _not_ use it for Production workloads. The release comes with several limitations:
+
+- The release is restricted to Free organizations. You will not be able to upgrade OrioleDB projects to larger instance sizes. If you want to run OrioleDB on a larger instance we suggest following the [Getting Started](https://www.orioledb.com/docs/usage/getting-started) guide on OrioleDB’s official website.
+- Index support is restricted to the Postgres default B-Tree index type. Other indexs like GIN/GiST/BRIN/Hash, and pgvector's HNSW/IVFFlat are not supported.
+- No support for Supabase Realtime
```
Comment (w3b6x9) [inline apps/www/_blog/2024-12-01-orioledb-launch.mdx:31]: @olirice:

```suggestion
- No support for Supabase Realtime Postgres Changes. Realtime Broadcast and Presence will continue to work.
```

**Pull request merged**